index.html

```diff
@@ -10,224 +10,293 @@
+                    </svg>
+         </a>
         </ul>
-        <button id="bttn">see project</button>
-    </section>
-    <section class="card_item">
-      <div class="snap">
-        <img src="images/snap3/Snapshoot Portfolio.png" alt="">
-      </div>
-      <h4 class="tex">Tonic</h4>
-      <ul class="list">
-        <li class="canopy"><b> CANOPY</b></li>
-        <li><img src="images/cano/Counter.png" alt=""></li>
-        <li class="bac">Back End Dev</li>
-        <li><img src="images/cano/Counter.png" alt=""></li>
-        <li class="bac">2015</li>
-      </ul>
-      <p class="p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-      <ul class="text">
-        <li class="lists">html</li>
-        <li class="lists">css</li>
-        <li class="lists">javascript</li>
-      </ul> 
-      <button id="bttn">see project</button>
-    </section>
-    <section class="card_item">
-      <div class="snap">
-        <img src="images/snap4/Snapshoot Portfolio.png" alt="">
-      </div>
-      <h5 class="tex"> Multi-Post Stories</h5>
-      <ul class="list">
-        <li class="canopy"><b> CANOPY</b></li>
-        <li><img src="images/cano/Counter.png" alt=""></li>
-        <li class="bac">Back End Dev</li>
-        <li><img src="images/cano/Counter.png" alt=""></li>
-        <li class="bac">2015</li>
-      </ul>
-      <p class="p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div> 
+</section>   
+<img src="eclipse/" alt="">
+</section>
+<section class="container">
+  <section class="card_item">
+    <div>
+      <img src="/images/Portoflio Card/Snapshoot Portfolio2.png" alt="">
+    </div>
+    
+    <h5 class="tex"> Tonic</h5>
+    <ul class="list">
+      <li class="canopy"><b> CANOPY</b></li>
+      <li><img src="images/cano/Counter.png" alt=""></li>
+      <li class="bac">Back End Dev</li>
+      <li><img src="images/cano/Counter.png" alt=""></li>
+      <li class="bac">2015</li>
+    </ul>
+    <p class="p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="text">
+      <li class="lists">html</li>
+      <li class="lists">css</li>
+      <li class="lists">javascript</li>
+    </ul>
+    <button id="bttn">see project</button>
```

Comment:
- [ ] There are a few changes to apply to this card. Kindly stick to the original Figma design.  Below are some of the suggestions but though not limited.
  - Kindly ensure that the image of the card is displayed. This can be achieved by using `./images` path. This is known as [`relative path`](https://www.w3schools.com/html/html_filepaths.asp) ☺️ .
  - The contents of the card should be displayed using [Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/). Kindly ensure that you apply this to make your project match the Figma design. 
      - NOTE: This should be applied to all the cards.
  - `CANOPY Back End Dev 2015` should be aligned on the same line with the `dots` directly in between. But according to your design, the dots are slightly up which mismatches the Figma design.
  - Kindly ensure to use the same `fonts` used in the Figma design.

- Your design 😎 
![card1](https://user-images.githubusercontent.com/99394374/213878947-b358e2f5-cf2e-4ae4-9ae6-de608e45b416.PNG)

- Figma design 😎 
![car1 fig](https://user-images.githubusercontent.com/99394374/213878964-52315ae3-c5d1-4fd7-b456-08efbe763273.PNG)